Swift/BRCryptoTests/BRCryptoCommonTests.swift

```diff
@@ -98,16 +99,44 @@ class BRCryptoCommonTests: XCTestCase {
         //
         // Phrase
         //
-        k = Key.createFrom (phrase: "nothing", words: BRCryptoAccountTests.words)
-        XCTAssertNil(k)
         k = Key.createFrom (phrase: "ginger settle marine tissue robot crane night number ramp coast roast critic", words: BRCryptoAccountTests.words)
         XCTAssertNotNil(k)
+        k = Key.createFrom (phrase: "ginger settle marine tissue robot crane night number ramp coast roast critic", words: nil)
```

Comment:
Shouldn't we make `word` a non-optional parameter?

Reply:
The idea was that you could install the words appropriate for your locale and then not worry about it again:
```
public final class Key {
    static public var wordList: [String]?
    // ...
    static public func createFrom (phrase: String, words: [String]? = wordList) -> Key? {
       // ...
```
Until the static `wordList` is assigned it will be `nil` and thus the optional argument, such as in `createFrom(phrase:words)`, must accept `nil`

Reply:
So it is a) provide `words`, b) assign `wordList` or c) handle `nil` (owing to no words or a corrupt phrase).